internal/form/question/service.go

```diff
@@ -0,0 +1,90 @@
+package question
+
+import (
+	"context"
+	databaseutil "github.com/NYCU-SDC/summer/pkg/database"
+	logutil "github.com/NYCU-SDC/summer/pkg/log"
+	"github.com/google/uuid"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/trace"
+	"go.uber.org/zap"
+)
+
+type Querier interface {
+	Create(ctx context.Context, params CreateParams) (Question, error)
+	Update(ctx context.Context, params UpdateParams) (Question, error)
+	Delete(ctx context.Context, params DeleteParams) error
+	ListByFormID(ctx context.Context, formID uuid.UUID) ([]Question, error)
+}
+
+type Service struct {
+	logger  *zap.Logger
+	queries Querier
+	tracer  trace.Tracer
+}
+
+func NewService(logger *zap.Logger, db DBTX) *Service {
+	return &Service{
+		logger:  logger,
+		queries: New(db),
+		tracer:  otel.Tracer("forms/service"),
+	}
+}
+
+func (s *Service) Create(ctx context.Context, input CreateParams) (Question, error) {
```

Comment:
Using FormRequest here breaks the separation of the service layer. In the future, if other services need to create forms, they would have to rely on a type defined in a handler, which isn’t ideal.

Please consider following the same approach as in the question service.